.github/workflows/appsec.yml

```
@@ -217,3 +217,17 @@ jobs:
       - if: always()
         uses: ./.github/actions/testagent/logs
       - uses: codecov/codecov-action@v3
+
+  propagation:
```

why call to this propagation, if we are testing lodash?

```suggestion
  lodash:
```

---

I called it `propagation` to intentionally group in one job all tests related to propagation of the libraries we support for that.

Are we fine grouping propagation tests in one single job or should we split them in one job per lib?

---

ok, but we can't test here propagation tests that are not related with externals libraries and are based on rewriter. They are executed with common appsec tests.